tests/test_integrated_channels/test_cornerstone/test_exporters/test_learner_data.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for Cornerstone Learner Data exporters.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import datetime
+import unittest
+
+import ddt
+import mock
+from freezegun import freeze_time
+from pytest import mark
+
+from django.utils import timezone
+
+from integrated_channels.cornerstone.exporters.learner_data import CornerstoneLearnerExporter
+from test_utils import factories
+from test_utils.fake_catalog_api import setup_course_catalog_api_client_mock
+
+
+@mark.django_db
+@ddt.ddt
+class TestCornerstoneLearnerExporter(unittest.TestCase):
+    """
+    Tests of CornerstoneLearnerExporter class.
+    """
+
+    NOW = datetime.datetime(2017, 1, 2, 3, 4, 5, tzinfo=timezone.utc)
+    NOW_TIMESTAMP = 1483326245000
```

Comment:
why can't we use `parse_datetime_to_epoch_millis` here to convert `NOW` into timestamp instead of hard coding timestamp here?